application/language/german/general_words_lang.php

```diff
@@ -7,9 +7,20 @@
 $lang['notice_turn_the_radio_on'] = 'Du hast heute noch keine QSOs gemacht; Zeit, das Funkgerät einzuschalten!';
 
 $lang['general_word_important'] = 'Wichtig';
+$lang['general_word_info'] = 'Info';
+$lang['general_word_choose_file'] = 'Datei wählen';
 
 $lang['general_word_date'] = 'Datum';
 $lang['general_word_time'] = 'Zeit';
+$lang['general_word_none'] = 'None';
+$lang['general_word_name'] = 'Name';
+$lang['general_word_location'] = 'Die Stelle';
+$lang['general_word_comment'] = 'Anmerkung';
+$lang['general_word_general'] = 'Allgemeines';
+$lang['general_word_satellite'] = 'Satelliit';
+$lang['general_word_satellite_short'] = 'Sat';
+$lang['general_word_notes'] = 'Anmerkungen';
```

Comment:
I don't know the context for this translation. Maybe "Hinweise" or "Notizen" could also be a good translation. Please can you provide a screenhot of it?

Reply:
You are right. I checked from my finnish translation and there it's more like notebook -kind of translation.

Reply:
OK. I've taken a look into my Cloudlog and "Notizen" makes more sense than "Anmerkungen".

Reply:
Tnx - "Notizen" makes sense in that context.